### DISCUSSION TITLE
File Winring0x64.sys

### DISCUSSION THREAD

**Discussion by Paneenutella0654** [seerge/g-helper#3070] [Q&A] [answered]
I have been undervolting on my pc with an amd cpu and these files were created, I read that they are needed for undervolting I was wondering if it was possible to move them to another location so they would not be on the desktop

**Comment by seerge** [comment, marked as answer]
@Paneenutella0654 hello, they are needed for undervolting and have to be in the same folder as exe. Just create a folder and move exe and those files there
